mlflow/store/artifact/cli.py

```diff
@@ -136,7 +136,7 @@ def download_artifacts(run_id, artifact_path, artifact_uri, dst_path):
     downloaded_local_artifact_location = _download_artifacts(
         artifact_uri=artifact_uri, run_id=run_id, artifact_path=artifact_path, dst_path=dst_path
     )
-    click.echo(downloaded_local_artifact_location)
+    click.echo(f"downloaded artifacts local location: {downloaded_local_artifact_location}\n")
```

Comment:
Why do we need this change?

Reply:
Because progress bar has output in stdout, we need this to capture the location correctly by pattern matching.

Reply:
I think we can check the final line in stdout.

Reply:
I think that would be not robust enough 🤔 If progress bar has no '\n' in line or if the behavior changes, and even if we print anything else into the stdout in the future, it's hard to know whether the last line is this file location or not.

Reply:
Can we add `\n` before `downloaded_local_artifact_location`?

Reply:
- keep in cli
- add new line click.echo("\n") before path echo
- revert this line change